packages/expansion-hub/src/errors/NackError.ts

```diff
@@ -0,0 +1,34 @@
+import {RevHubError} from "./RevHubError";
+
+//Taken from https://github.com/apollographql/invariant-packages/blob/779f525d6b475653050cb17976753de287e30f91/packages/ts-invariant/src/invariant.ts#L2-L7
+/*
+Copyright (c) 2019 Apollo GraphQL
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+---
+
+Modified to use Object.setPrototypeOf instead of __proto__
+
+ */
+export const {
+    setPrototypeOf = function (obj: any, proto: any) {
+        Object.setPrototypeOf(obj, proto);
+        return obj;
+    },
+} = Object;
```

Comment:
Why is adding a `setPrototypeOf()` method to `Object` necessary? (I may be misunderstanding what's going on there). It seems to me like this is all we need, right?

```suggestion
function setPrototypeOf(obj: any, proto: any) {
    Object.setPrototypeOf(obj, proto);
    return obj;
}
```